homeassistant/components/homekit/type_covers.py

```diff
@@ -119,7 +119,7 @@ def move_cover(self, value):
     def update_state(self, new_state):
         """Update cover position after state changed."""
         current_position = new_state.attributes.get(ATTR_CURRENT_POSITION)
-        if isinstance(current_position, int):
+        if isinstance(current_position, (float, int)):
             self.char_current_position.set_value(current_position)
```

Comment:
Does this call support floats or do we need to cast it ?

Reply:
That's good notice. I checked Apple's specs, and I think we need to cast it here anyway.
<img src="https://user-images.githubusercontent.com/15189627/72974191-e2738400-3dce-11ea-94c9-243bb3f1ba9a.png" width="500">

I've pushed the fix in a separate commit. Thanks a lot for your review.